src/hooks/sdk/useSelectAudioOutputDevice.tsx

```diff
@@ -1,16 +1,18 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { DefaultBrowserBehavior } from 'amazon-chime-sdk-js';
 import { useCallback } from 'react';
 
 import { useMeetingManager } from '../../providers/MeetingProvider';
-import { supportsSetSinkId } from '../../utils/device-utils';
 
-export const useSelectAudioOutputDevice = () => {
+export const useSelectAudioOutputDevice = (): ((
+  deviceId: string
+) => Promise<void>) => {
   const meetingManager = useMeetingManager();
 
   const selectDevice = useCallback(async (deviceId: string) => {
-    if (supportsSetSinkId()) {
+    if (new DefaultBrowserBehavior().supportsSetSinkId()) {
```

Comment:
Looks like we do use `new DefaultBrowserBehavior().supportsSetSinkId()` now at multiple places so good to keep the device-utils?

Reply:
It's a TODO item to remove it.
```tsx
// TODO: Remove this and use DefaultBrowserBehavior.supportsSetSinkId from JS SDK v2.x
export const supportsSetSinkId = (): boolean => {
  return 'setSinkId' in HTMLAudioElement.prototype;
};
```

Reply:
I meant, we can still keep the util and move `new DefaultBrowserBehavior().supportsSetSinkId()` under it over writing the whole again each time we want to check, no strong opinion though, up to you.